ble/UUID.h

```diff
@@ -42,7 +97,7 @@ class UUID {
      *         different service or characteristics on the BLE device.
      *
      * @param longUUID
-     *          The 128-bit (16-byte) UUID value, MSB first (big-endian).
+     *          The 128-bit (16-byte) UUID value, LSB first (little-endian).
```

Comment:
Happy with having an additional parameter. And I agree with @pan- that users shouldn't be depending on the internal representation of the long UUID, except when they either compare or when they need to populate an advertisement payload. We already provide a comparison operator. We could also provide an helper to accumulate UUID in the adv. payload.